supabase/migrations/25_fix_billing_security.sql

```diff
@@ -0,0 +1,166 @@
+begin;
```

Comment:
I don't think we need to explicitly begin/commit transactions in migrations, since the supabase cli would do that already. Unless I'm missing something?

Reply:
Oh, I was just copying the other migrations that seemed to all say 
```sql
-- Always use a transaction, y'all.
begin;
```
at the top, though I am seeing some that don't have it... do you feel strongly that it should be removed? If not I think it's probably marginally safer to keep it

Reply:
I usually apply them with  psql, so it's good practice imo. Postgres doesn't mind if they're nested. Also I'm 70% that supabase doesn't wrap in a txn.